server/grpc_service.go

```diff
@@ -522,6 +522,14 @@ func (s *GrpcServer) PutStore(ctx context.Context, request *pdpb.PutStoreRequest
 	log.Info("put store ok", zap.Stringer("store", store))
 	CheckPDVersion(s.persistOptions)
 
+	go func(url string) {
+		// tikv may not ready to server.
```

Comment:
What happens to down tikv?

Reply:
Keeping  the config of all the tikv  to be same in the cluster is the precondition，so the pd should be sync the tikv config even through some tikv is down.  if the precondition is not exist, the cluster will work well in past.
 In upgrade, the configuration of the TiKV may be different,syncing with tikv is ok in the end.

Reply:
Does TiFlash have the status API too?

Reply:
it is a good question, it should ignore tiflash.